akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala

```diff
@@ -649,6 +649,29 @@ private[akka] trait RememberingShard {
   }
 }
 
+/** INTERNAL API */
+private[akka] object PersistentShard {
```

Comment:
`@InternalApi` also

Reply:
pro tip: add an IntelliJ code template for:
```
    /**
     * INTERNAL API
     */
    @InternalApi private[akka] 
```